CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/value/expression/ExpressionCompiler.java

```diff
@@ -100,7 +116,9 @@ public static Object compileSequenceExpression(SequenceExpression expression, Va
                 final String operatorName = CONVERTER_CLASSES.get(operator.getOperator());
                 final Object result = operators.getRegistry(operatorName).apply(right);
                 if (result == null) {
-                    throw new ExpressionException("Unable to find implementation of operator: " + operatorName + " (" + right.getClass().getCanonicalName() + ")");
+                    Platform.runLater(() -> NotifyDisplayer.displayAlert(QUERY_ERROR, OPERATOR_NOT_FOUND, 
+                            String.format(OPERATOR_ERROR_MSG, operatorName), Alert.AlertType.ERROR));
+                    LOGGER.log(Level.WARNING, OPERATOR_NOT_FOUND_MSG, operatorName);
```

Comment:
This is a cool way of doing the alert and logging the exception. I would like to do this across the board. :+1: